server/views/components/copy-url/copy-url.njk

```diff
@@ -1 +1,7 @@
-<button data-component-name="Copy URL button" data-component-id="{{ model.id }}" data-track-click="{}" class="btn btn--dark is-hidden js-copy-url">Copy URL</button>
+<div class="flex">
+  <input style="flex-grow: 1;" type="text" value="{{ model.url }}" />
+  <button
+    data-copy-text="{{ model.url }}"
+    class="btn btn--dark is-hidden js-copy-url"
+    style="width: auto;">Copy link</button>
```

Comment:
We should address the button having `width: 100%` set on it, perhaps. Maybe add a TODO to get rid of this inline style?